File: TecdamExemploHttp/app/src/main/java/br/com/dfn/exemplohttp/communication/ServiceClient.java
package br.com.dfn.exemplohttp.communication;

import retrofit2.Retrofit;
import retrofit2.converter.gson.GsonConverterFactory;

public class ServiceClient {

    private static String URL_BASE =
            "https://newsapi.org/v1/";

    public static Retrofit getRetrofit() {
        return new Retrofit.Builder()
                .baseUrl(URL_BASE)
                .addConverterFactory(GsonConverterFactory.create())
                .build();
    }


}

File: kotlinWithRetrofit/app/src/main/java/br/com/dfn/koltinwithretrofit/model/remote/ServiceClient.kt
package br.com.dfn.koltinwithretrofit.model.remote

import retrofit2.Retrofit
import retrofit2.converter.gson.GsonConverterFactory

class ServiceClient {
    companion object {
        val retrofit = Retrofit.Builder()
                .baseUrl("https://swapi.co/api/")
                .addConverterFactory(GsonConverterFactory.create())
                .build()
    }
}